ansible.cfg

```diff
@@ -1,17 +0,0 @@
-# see http://docs.ansible.com/intro_configuration.html
-[defaults]
-
-# Do not show skipped hosts
-display_skipped_hosts = False
-
-# Do not use cowsay
-nocows = 1
-
-# Connect to up to 50 server in parallel
-forks = 50
-
-# Speed improvement, but may break sudo with requiretty
-pipelining = True
```

Comment:
Why ?

Reply:
Not used anymore. Molecule manages its own ansible.cfg. we can pass params through molecule.yaml files